google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -28,10 +28,13 @@
 from google.cloud.firestore_v1beta1 import types
 from google.cloud.firestore_v1beta1.field_path import FieldPath
 from google.cloud.firestore_v1beta1.field_path import parse_field_path
-from google.cloud.firestore_v1beta1.gapic import enums
-from google.cloud.firestore_v1beta1.proto import common_pb2
-from google.cloud.firestore_v1beta1.proto import document_pb2
-from google.cloud.firestore_v1beta1.proto import write_pb2
+
+# from google.cloud.firestore_v1beta1 import DocumentTransform
+from google.cloud.firestore_v1beta1.types.write import DocumentTransform
```

Comment:
I think I'm missing something, because I don't understand the importance of the comment here.